website/content/en/docs/upgrading/compatibility.md

```diff
@@ -15,9 +15,9 @@ Before you begin upgrading Karpenter, consider Karpenter compatibility issues re
 
 [comment]: <> (the content below is generated from hack/docs/compataiblitymetrix_gen_docs.go)
 
-| KUBERNETES |  1.23   |  1.24   |  1.25   |  1.26   |  1.27   |  1.28   |  1.29   |
-|------------|---------|---------|---------|---------|---------|---------|---------|
-| karpenter  | 0.21.x+ | 0.21.x+ | 0.25.x+ | 0.28.x+ | 0.28.x+ | 0.31.x+ | 0.34.0+ |
+| KUBERNETES | 1.23     | 1.24     | 1.25     | 1.26     | 1.27     | 1.28     | 1.29       |
```

Comment:
This data is generated. If this is a change that we want to make, I think we need to make a change to https://github.com/aws/karpenter-provider-aws/tree/main/tools/kompat to generate different data

Reply:
reverted back the doc and committed a fix to kompat